tests/test_model.py

```diff
@@ -3572,3 +3572,482 @@ def test_add_boundary_condition_on_polygon(self, create_default_3d_soil_material
             assert generated_point_id == expected_point_id
             assert generated_point.id == expected_point.id
             npt.assert_allclose(generated_point.coordinates, expected_point.coordinates)
+
+    def test_get_bounding_box_soil(self, create_default_3d_soil_material: SoilMaterial):
+        """
+        Test if the bounding box of all soil volumes is correctly calculated. A model is created with a soil layer and
+        the bounding box of the soil volume is checked.
+
+        Args:
+            - create_default_3d_soil_material (:class:`stem.soil_material.SoilMaterial`): default soil material
+        """
+        ndim = 3
+
+        layer_coordinates = [(0, 0, 0), (1, 0, 0), (1, 2, 3), (0, 2, 3)]
```

Comment:
i think these coordinates are a bit weird together with the extrusion, maybe leave the z-coord at 0, and only generate through extrusion

Reply:
changed in all tests